armi/settings/fwSettings/globalSettings.py

```diff
@@ -406,18 +408,37 @@ def defineSettings() -> List[setting.Setting]:
         ),
         setting.Setting(
             CONF_BU_GROUPS,
-            default=[10, 20, 30, 100],
-            label="Burnup Groups",
+            default=[10, 20, 30],
```

Comment:
i changed both these to have max implied values of infinite  since temperature really has no physical upper bound like burnup. should be easier for users since they dont have to explicitly add 100.

Reply:
Will users have to change all existing input files to remove the "100"? This might be more confusing.

Reply:
no. its just adding inf to the end. input change is not required. if they didnt change they would have empty group between 100 and finity, which isn't really a problem. I think implied upper bound inf on temp is the right thing. i can change this back if you like